elcid/migrations/0030_auto_20181123_0849.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
@fredkingham This migration is back... `git add .` again ? ;)

Reply:
👍 removed.